packages/sdk/src/interfaces/llm.ts

```diff
@@ -1,12 +1,15 @@
 import { InterfaceDeclaration } from '../integration/definition'
-import { z } from '../zui'
+import z from '../zui'
 
 const ToolCallSchema = z.object({
   id: z.string(),
   type: z.enum(['function']),
   function: z.object({
     name: z.string(),
-    arguments: z.string(),
+    arguments: z
```

Comment:
**chore:** this is a breaking change so, the backend will complain and throw a 4xx HTTP error code when deploying. To prevent this we need to bump:
1. all llm integrations by a **major**
2. the llm interface by any jump

Reply:
Done in https://github.com/botpress/botpress/pull/13159/commits/a352166217ccc2c75fda642e20b9acb4d905feba and https://github.com/botpress/botpress/pull/13159/commits/424a05c6a7d843c9ce3c46fa1cbde0be22e51ec3